lib/regex.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" ?>
+<?xml-stylesheet type='text/xsl' href='interface.xsl'?>
+<interface uri="http://repo.roscidus.com/lib/regex" xmlns="http://zero-install.sourceforge.net/2004/injector/interface" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://zero-install.sourceforge.net/2004/injector/interface http://0install.de/schema/injector/interface/interface.xsd http://0install.de/schema/desktop-integration/capabilities http://0install.de/schema/desktop-integration/capabilities/capabilities.xsd">
+  <name>Regex</name>
+  <summary xml:lang="en">Regex: searches for and matches text strings</summary>
+  <description xml:lang="en">A regular expression (or regexp, or pattern) is a text string that describes some set of strings. 
+
+Using the Regex library, you can: 
+•see if a string matches a specified pattern as a whole, and 
+•search within a string for a substring matching a specified pattern. 
+
+Regex provides three groups of functions with which you can operate on regular expressions. One group--the GNU group--is more powerful but not completely compatible with the other two, namely the POSIX and Berkeley UNIX groups; its interface was designed specifically for GNU. The other groups have the same interfaces as do the regular expression functions in POSIX and Berkeley UNIX. 
+</description>
+  <homepage>http://www.gnu.org/directory/regex.html</homepage>
+  <implementation arch="Windows-*" id="sha1new=ccdcdaf83f133fdec116d6f35a76ab74734eaedf" license="LGPL (GNU Lesser General Public License)" released="2007-10-24" version="2.7">
```

Comment:
Best to use `arch="Windows-i486"` for 32-bit DLLs.

Reply:
Done